src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -179,12 +179,26 @@ private void applyStringFilter(String filterString) {
             } else {
                 this.applyFilterExpression(Qualifier.EMPTY);
             }
-        } catch (ParseException ex) {
+        } catch (FilterException ex) {
             this.applyFilterExpression(Qualifier.EMPTY);
             // Overrides message in status bar
-            UI.status.displayMessage("Panel " + (panelIndex + 1)
-                + ": Parse error in filter: " + ex.getMessage());
+            UI.status.displayMessage(resolvePanelName(
+                panelMenuBar.getPanelName(), PanelMenuBar.DEFAULT_PANEL_NAME, 
+                panelIndex) + ": " + ex.getMessage());
+        }
+    }
+
+    /**
+     * Appends panel index to panel name if a panel is unnamed.
+     * This allows user to identify an otherwise unnamed panel.
+     * @param panelName
+     * @return final panel name shown to users
+     */
+    private String resolvePanelName(String panelName, String defaultName, int panelIndex) {
```

Comment:
I think removing the default name which is a constant and panelIndex which belongs to the class is fine. However, keeping the panelName as parameter because it is not a direct property of the class